src/VisualStudio/IntegrationTest/TestUtilities/OutOfProcess/VisualStudioWorkspace_OutOfProc.cs

```diff
@@ -20,12 +21,21 @@ public bool IsUseSuggestionModeOn()
         public void SetUseSuggestionMode(bool value)
             => _inProc.SetUseSuggestionMode(value);
 
+        public void SetOptionInfer(bool value)
+            => _inProc.SetOptionInfer(value);
+
+        public void SetPersistenceOption(bool value)
+            => SetPerLanguageOption("Enabled", "FeatureManager/Persistence", null, value ? "true" : "false");
```

Comment:
Don't we already have internals visible to Roslyn to use the IOptionService? Can we just reference the option directly?